examples/ct/ct_handlers.go

```diff
@@ -202,3 +340,68 @@ func (c CTRequestHandlers) RegisterCTHandlers() {
 	http.HandleFunc(pathFor("get-roots"), wrappedGetRootsHandler(c.trustedRoots, c.rpcClient))
 	http.HandleFunc(pathFor("get-entry-and-proof"), wrappedGetEntryAndProofHandler(c.rpcClient))
 }
+
+// Generates a custom error page to give more information on why something didn't work
+// TODO(Martin2112): Not sure if we want to expose any detail or not
```

Comment:
If `err != nil`, is `isPrecert` still meaningful? The following logging implies it is.